vip-plugins/vip-plugins.php

```diff
@@ -0,0 +1,222 @@
+<?php 
+
+/**
+ * Retrieve featured plugins from the vip.wordpress.com API
+ *
+ * @return array an array of plugins
+ */
+function wpcom_vip_fetch_vip_featured_plugins() {
+	$plugins = wp_cache_get( 'wpcom_vip_featured_plugins' );
+
+	if ( false === $plugins ) {
+		$plugins = array();
+		$url_for_featured_plugins = 'https://vip.wordpress.com/wp-json/vip/v1/plugins?type=technology';
+		$response = vip_safe_wp_remote_get( $url_for_featured_plugins, false, 3, 5 );
+
+		if ( ! $response || is_wp_error( $response ) ) {
+			trigger_error( 'The API on vip.wordpress.com is not responding (' . esc_url( $url_for_featured_plugins ) . ')' );
```

Comment:
We also need to cache an error or a non-`false` value here to avoid breaking things when the VIP site isn't available. Can cache it for a shorter period (e.g. `MINUTE_IN_SECONDS`).